logstash-core/src/test/java/org/logstash/config/ir/compiler/OutputDelegatorTest.java

```diff
@@ -245,3 +245,13 @@ private long getMetricLongValue(RubyHash metricStore, String symbolName) {
         return count.getLongValue();
     }
 }
+
+class StrategyPair {
```

Comment:
@danhermann LGTM now thank, but if you have another 60s here, make this a `private static` nested class instead of two top level classes in one file ? :D